cloud/integrations.md

```diff
@@ -0,0 +1,137 @@
+import ExporterRegionNote from 'versionContent/_partials/_cloud-integrations-exporter-region.mdx';
+
+# Integrate Timescale Cloud services with third-party monitoring tools
+
+You can export your service telemetry to a third-party monitoring tool, such as
+[Datadog][datadog] or [AWS CloudWatch][cloudwatch]. Exported metrics include
+CPU usage, RAM usage, and storage.
+
+## Export telemetry data to Datadog
+
+Export telemetry data to Datadog by:
+
+1.  Creating a data exporter
+1.  Attaching your database service to the exporter
+
+<ExporterRegionNote />
+
+<procedure>
+
+### Creating a data exporter for Datadog
+
+1.  In the Timescale Cloud console, navigate to `Integrations`.
+1.  Click `Create exporter`.
+1.  Under `Choose a provder`, choose `Datadog`.
+1.  Choose an AWS region for your exporter to live within Timescale Cloud. The
+    exporter is only available to database services in the same AWS region.
+1.  Name your exporter. This name appears in the Cloud console, so choose a
+    descriptive name.
+1.  Add a Datadog API key. If you don't have an API key yet, you can create one
+    by following the instructions in the [Datadog documentation][datadog-docs].
+1.  Under Site, choose your Datadog region. You can choose a region to meet any
+    regulatory requirements or application needs you might have.
+1.  Click `Create endpoint`.
+
+<img class="main-content__illustration"
+src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-integrations-datadog.png"
+alt="Screenshot of the menu for adding a Datadog exporter" />
+
+</procedure>
+
+<AttachExporter />
+
+## Export telemetry data to AWS CloudWatch
+
+Export telemetry data to AWS CloudWatch by:
+
+1.  Creating a data exporter
+1.  Attaching your database service to the exporter
+
+<ExporterRegionNote />
+
+<procedure>
+
+### Creating a data exporter for AWS CloudWatch
+
+1.  In the Timescale Cloud console, navigate to `Integrations`.
+1.  Click `Create exporter`.
+1.  Under `Choose a provder`, choose `AWS CloudWatch`.
+1.  Choose an AWS region for your exporter to live within Timescale Cloud. The
+    exporter is only available to database services in the same AWS region.
+1.  Name your exporter. This name appears in the Cloud console, so choose a
+    descriptive name.
+1.  Define names for your CloudWatch log group, log stream, and namespace. If
+    you're uncertain, use the default values. For more information on naming log
+    groups and logs streams, see [the AWS CloudWatch
+    docs][cloudwatch-log-naming].
+1.  Enter your AWS credentials. To get your AWS keys, you need to create a new
+    Identity and Access Management (IAM) user in your AWS console. Make sure
+    your new user has restricted access to only Cloudwatch, and keep your keys
+    secret. For instructions, see the [AWS documentation][aws-access-keys].
+1.  Select an AWS Region for your CloudWatch instance.
+1.  **OPTIONAL** Define an IAM role if you want to upload metrics to another
```

Comment:
possible to change this to something along the lines of:
"Optional: IAM role to use for uploading metrics. Having a dedicated role with only CloudWatch permissions is a recommended security practice."